### DISCUSSION TITLE
Connecting with router (x86)

### DISCUSSION THREAD

**Discussion by evanrodgers** [SmoothWAN/SmoothWAN#18] [General & Help]
Hi there,

I installed SmoothWAN on a generic micro PC and it's working great for the most part. However, when I try to connect to the Wi-Fi router (via ethernet), it won't broadcast the internet, but I can access the SmoothWAN web interface via the Wi-Fi router's LAN.

I am logged into Speedify and that part is working correctly.

Basically, how do I get SoothWAN to output the internet to the Wi-Fi router?

**Comment by TalalMash** [comment]
Hello!

While the original instructions are plug and play, few products have limited DHCP settings on the WAN side, I will change the diagrams and instructions for bridged configuration as its is more compatible and appropriate albeit requiring few extra steps.

1. Login to the Wi-Fi router's admin page or phone app.
2. Set "LAN IP address" to `172.17.17.5` (you can change the last number by preference from 5-10)
3. Disable "DHCP Server"
4. Now plug SmoothWAN appliance to the LAN port instead of WAN.

You have just created a "Bridged Access Point".
You can access the Wi-Fi router setup page from `172.17.17.5`

Few examples with common vendors:
TP-Link
![image](https://user-images.githubusercontent.com/96490382/162148977-397dfdae-56b8-43ae-8314-72eb1a3088cd.png)
Asus
![image](https://user-images.githubusercontent.com/96490382/162149499-ec290f69-8cf0-4de5-827f-66d772110eac.png)
![image](https://user-images.githubusercontent.com/96490382/162149580-8745d63f-3bb2-4b5d-9aca-755a06f2dd61.png)
Linksys
![image](https://user-images.githubusercontent.com/96490382/162150449-d08b8244-c984-4b80-8809-9157e2533abb.png)
Netgear
![image](https://user-images.githubusercontent.com/96490382/162150782-8f989520-31e6-42b7-908d-72990da3ebe5.png)
Tenda
![image](https://user-images.githubusercontent.com/96490382/162151069-23342150-3f05-4dc0-8efe-ff7b666298c6.png)

**Reply by evanrodgers** [reply]
Okay, that makes sense, thank you!!

**Comment by blaircullen** [comment]
Hi have what is most likely a very silly question, but it has stumped me nonetheless.  I have the "LAN2" and "WAN" ports connected to external cellular modems which are being bonded beautifully by Speedify using my Slate AX.

I'd like to directly connect via ethernet the speedify bonded output to my Unifi Dream Machine SE WAN2 port as a failover.  Can you tell me the best way to do this?

**Reply by blaircullen** [reply]
I should add, I can't seem to get the "LAN1" port on the Slate AX to actually do anthing.

**Reply by TalalMash** [reply]
Hi! That's odd, do you have any IP address when connected via LAN1 port?

**Reply by blaircullen** [reply]
No I can't seem pull an IP, what is the name of the interface I'm looking for for LAN1?  Is that eth0?

**Reply by blaircullen** [reply]
![screencap](https://user-images.githubusercontent.com/20569385/211363493-3e7b6ff1-0ac7-4813-b7d8-b3ddf5963309.png)

**Reply by TalalMash** [reply]
Slate AX by default bridgle LAN port is eth2:
![image](https://user-images.githubusercontent.com/96490382/211369814-8d935fa0-9def-400e-ad53-551b608d2d97.png)

**Reply by blaircullen** [reply]
Hi there, I just reset the Slate AX and do have what you posted above with bridge port eth2 being br-lan.  However when I plug anything into the LAN1 port I don't get an IP, in fact, on most devices I've tried it says "no cable connected."  Do I need to add a static IP or is there DHCP that will hand out an IP on eth2?  Just at a loss of what might be happening.

**Reply by TalalMash** [reply]
That is weird, is it doing the same with stock firmware? no cable connected suggests hardware issue usually.

**Comment by blaircullen** [comment]
My thought exactly, I’m not a networking Jedi master, but I’m also not a newbie, this has really had me scratching my head.  Gonna order another one from Amazon and see if the new one has the same issue.  Thank you so much for your help troubleshooting this!
